src/lib.rs

```diff
@@ -39,6 +39,9 @@ use log::Log;
 #[cfg(test)]
 use log::*;
 
+#[cfg(feature = "paris")]
+mod paris;
```

Comment:
Please make the module public, so crates with edition 2018 or newer, can import the macros directly via `use` and not only via `#[macro_use]`.

Reply:
Not sure If I did this correctly... please take a look at my additional commits.